dotCMS/src/curl-test/Permission_Resource.postman_collection.json

```diff
@@ -91,7 +103,34 @@
 									"});",
 									"pm.test(\"Icon should be 360\", function() {",
 									"    pm.expect(contentType.icon).to.eql(\"360\");",
-									"});"
+									"});",
+									"",
+-									"pm.test(\"Content type fields have expected structure\", function () {",
+-									"  const createdContentType = pm.response.json().entity[0];",
```

Comment:
These are removed?

Reply:
same thing, unable to import into postman, because have incorrect format